### ISSUE TITLE
Implement FromIterator on Sentence

### ISSUE THREAD

**Issue by sebpuetz** [danieldk/conllx-rs#12]
While writing a conversion method to `Sentence` I thought it'd be nice to just call 
```Rust
let sentence = tokens.into_iter().collect::<Sentence>();
```
rather than writing
```Rust
let mut sentence = Sentence::new();
for token in tokens {
    sentence.push(token);
}
```

**Comment by danieldk** [comment]
Go for it! :)

**Comment by danieldk** [comment]
Merged in 30792fcb4779433c8965c03ad50f76f313361d28.
